common/openapi.yml

```diff
@@ -1,12 +1,27 @@
 openapi: 3.0.3
 info:
   title: Terra Batch Analysis
-  description: Manage batch analyses, workflows, and submissions in userspace
+  description: Manage batch analyses, run configurations, and run sets in userspace
   version: 0.0.1
 servers:
   - url: ./
     description: Relative to the current swagger page
 paths:
+  /pets:
+    patch:
+      tags: [ public ]
+      operationId: getPets
```

Comment:
Is this intentional? 🤔

Reply:
Oops, no, this was left over testing the `oneOf` examples from their docs 🤦